File: AntiClown.Web.Api/Dto/F1Predictions/F1StandingsRowDto.cs
using AntiClown.Entertainment.Api.Dto.F1Predictions;

namespace AntiClown.Web.Api.Dto.F1Predictions;

public record F1StandingsRowDto
{
    public Guid UserId { get; set; }
    public int TotalPoints { get; set; }
    public F1PredictionUserResultDto?[] Results { get; set; }
}

File: src/Web/AntiClown.Web.Front/src/Dto/F1Predictions/F1StandingsRowDto.ts
import { F1PredictionUserResultDto } from "./F1PredictionUserResultDto";
import { F1PodiumDto } from "./F1PodiumDto";

export interface F1StandingsRowDto {
  userId: string;
  totalPoints: number;
  results: (F1PredictionUserResultDto | null)[];
  previousPodiums: F1PodiumDto[];
}
